helper-scripts/options/balena.json

```diff
@@ -0,0 +1,30 @@
+{
+  "modelName": "balena",
+  "numConsecutiveDays": 5,
+  "supportStartHour": 6,
+  "supportEndHour": 26,
+  "slotsInDay": 26,
+  "shiftMinDuration": 2,
+  "shiftMaxDuration": 8,
+  "optimizationTimeout": 3600,
+  "specialAgentConditions": {
+    "agentsWithMaxHoursShift": [
+      {
+        "handle": "@samothx",
+        "value": 4
+      }
+    ],
+    "agentsWithMinHoursWeek": [
+      {
+        "handle": "@saintaardvark",
```

Comment:
Will no longer be part of the company starting next week, just a note.

Reply:
yep, removed already. PR was older